tests/resources/test_material_run.py

```diff
@@ -164,6 +165,33 @@ def test_filter_by_name(collection, session):
     assert sample_run['uids'] == runs[0].uids
 
 
+def test_list_by_name(collection, session):
```

Comment:
It would be nice to also get https://github.com/CitrineInformatics/citrine-python/pull/145 in

Reply:
Backend changes not present, disregard. Thanks for catching @kyle-laplante